src/paragraph.js

```diff
@@ -75,6 +73,14 @@ export default class Paragraph extends Plugin {
 		}, { priority: 'low' } );
 
 		editor.commands.set( 'paragraph', new ParagraphCommand( editor ) );
+
+		// Post-fixer that takes care of adding empty paragraph elements to empty roots.
+		doc.on( 'change', ( evt, type, changes, batch ) => findEmptyRoots( doc, batch ) );
+		doc.on( 'changesDone', autoparagraphEmptyRoots, { priority: 'lowest' } );
+		editor.on( 'dataReady', () => {
```

Comment:
I'd love to avoid relying on the editor event here... This should be engine only behaviour. The less layers it crosses the safer it is.

As @scofalik told me this is due to the fact that some base editor classes don't even call `setData()` initially. I think that we don't have to create those paragraphs if an editor didn't load any data (i.e. `setData()` wasn't called at all).

Now, the question is whether `setData('')` cause any `#change` to be fired...